### DISCUSSION TITLE
Microsoft/MS Windows 10 release (compatility)

### DISCUSSION THREAD

**Discussion by DoS007** [linuxmint/discussions#652] [Ideas]
Would be nice, if one could use nemo in windows 10.

Dolphin e.g. has releases for it, see https://cdn.kde.org/ci-builds/system/dolphin/master/windows/ .

**Comment by JosephMcc** [comment]
Sorry, but this isn't happening. Nemo is meant for Linux and there is no desire to work on this.

**Reply by DoS007** [reply]
Ok, thanks for the feedback.
